release/release.md

```diff
@@ -0,0 +1,112 @@
+# How to do a Kata Containers Release
+
+Hi if you are reading this document you may also want to create a kata Containers Release.
+
+The Kata Containers Release Process is defined in the follwoing documents.
+
+https://github.com/kata-containers/documentation/blob/master/Releases.md
+
+To simply the process of read each release we have created a checklist for it.
+
+https://github.com/kata-containers/documentation/blob/master/Release-Checklist.md
+
+In order to simplify the process of do the Release Checklist we have semi-automated most of the process.
+
+So lets get started.
+
+## Requeriments 
+- Go (https://golang.org/doc/install#)
+
+- hub (https://github.com/github/hub)
+  If you are using github 2FA, configure hub manually creating a file `~/.config/hub`
+  ```yaml
+  github.com:
+  - user: YOUR_USER
+  oauth_token: TOKEN
+  protocol: https
+  ```
+  - OBS account with permissions on /home:katacontainers (https://build.opensuse.org/project/subprojects/home:katacontainers)
+  
+  - Github permissions to push tags an creates Releases in Kata repositories.
+  
+  - GPG configured to sign git tags. https://help.github.com/articles/generating-a-new-gpg-key/
+  
+  - Your github should be configured to use your ssh keys (to push to branches). See https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/.
+    * As alternative you can configure hub to do push and forks with https, `git config --global hub.protocol https` (Not tested yet) *
+  
+
+## Get packaging kata repository.
+```bash
+go get -d github.com/kata-containers/packaging
+```
+
+# Start release process
+```bash
+cd ${GOPATH}/src/github.com/kata-containers/packaging
+#make sure you are up-to-date.
+git pull
+```
+1. Bump repositories
+```bash
+cd ${GOPATH}/src/github.com/kata-containers/packaging/release
+export NEW_VERSION=X.Y.Z
+export BRANCH="master"
+./update-repository-version.sh -p ksm-throttler "$NEW_VERSION" "$BRANCH"
+./update-repository-version.sh -p proxy "$NEW_VERSION" "$BRANCH"
+./update-repository-version.sh -p shim "$NEW_VERSION" "$BRANCH"
+./update-repository-version.sh -p runtime "$NEW_VERSION" "$BRANCH"
+./update-repository-version.sh -p osbuilder "$NEW_VERSION" "$BRANCH"
+./update-repository-version.sh -p agent "$NEW_VERSION" "$BRANCH"
+```
+
+The commands from above will create a github pull request in the Kata projects.
+Work with the Kata approvers to verify that the CI works and the PR are merged.
+
+2. Create Github tags
```

Comment:
Just a note - you can use '1' over and over again in a number list and it should sequentially number itself. This helps a lot later on if you need to add or delete an entry, as all entries are labelled '1' in the source.

Reply:
I tried but I did not work, I wonder if gihub does not detect for the different paragraphs between list.